app/assets/javascripts/sequence_show.js

```diff
@@ -1,7 +1,10 @@
 function init_sequence_show(data) {
 
     // set up the fancy tree
-    initLineageTree(data.tree);
+    initD3TreeView(data.tree, "#lineage-tree");
```

Comment:
This works fine, both trees are created with the new D3TreeView


_[Original comment](https://github.ugent.be/unipept/unipept/pull/528#discussion_r3467) by calmalak on Mon Nov 09 2015 at 22:06._